src/en/manganelo/build.gradle

```
@@ -2,8 +2,8 @@ ext {
     extName = 'Manganato'
     extClass = '.Manganato'
     themePkg = 'mangabox'
-    baseUrl = 'https://manganato.com'
-    overrideVersionCode = 2
+    baseUrl = 'https://www.natomanga.com'
+    overrideVersionCode = 3
```

```suggestion
    overrideVersionCode = 2
```

... and all other comments for Mangakakalot applies for Manganato too.